Makefile

```
@@ -132,7 +132,6 @@ CORE_DEPENDS_amd64?=	beep \
 # transition helpers for PHP 8/Phalcon 5 migration
 CORE_DEPENDS_PHP74=	php74-json php74-openssl php74-phalcon${CORE_PHALCON}
```

```suggestion
CORE_DEPENDS_PHP74=	php74-json php74-openssl php74-phalcon
```

to align with below change